recipes/default.rb

```diff
@@ -28,6 +28,16 @@
   end
 end
 
+# Create the mongodb user if not exist
+user node['mongodb3']['user'] do
```

Comment:
@barata 
the package installs the default user for mongodb like ; ubuntu : mongod, RHEL : mongod.
for that, I have some conditional attribute on default attribute

```
# Default attribute for MongoDB installation
case node['platform_family']
  when 'rhel', 'fedora'
    mongo_user = 'mongod'
    mongo_group = 'mongod'
    mongo_dbpath = '/var/lib/mongo'
    mongo_pid_file = '/var/run/mongodb/mongodb.pid'
    config_processManagement_fork = true
  when 'debian'
    mongo_user = 'mongodb'
    mongo_group = 'mongodb'
    mongo_dbpath = '/var/lib/mongodb'
    mongo_pid_file = nil
    config_processManagement_fork = nil
end
```
so, I would strongly recommend do not set the attribute value for `['mongodb3']['user']` and `['mongodb3']['group']`

can you remove the user and group creating part from your PR please?